Add new info about custom responder types (#15701)
DataDog/documentation: content/en/monitors/incident_management/incident_details.md

As work for different tasks is finished, individual tasks can be marked as completed by clicking the checkbox to the left of the task's description. If you have a large number of tasks, you can filter them down by searching for key words or by hiding completed tasks from view.
  
## Response Team section

<div class="alert alert-warning">
This feature is in open beta.
</div>

{{< img src="monitors/incidents/incident_response_team.png" alt="The Response Team section of the Incident Details page for a specific incident" style="width:80%;">}}

In the Response Team section, you can form your response team by adding other users and assigning them roles to carry out in the process of resolving an incident. The two default responder types provided by Datadog are:

1. `Incident Commander` - The individual responsible for leading the response team 
3. `Responder` - An individual that actively contributes to investigating an incident and resolving its underlying issue

> If you wish to create custom responders roles, you can do so in the [Incident Settings for Responder Types][15]. This allows you to create new responder types with custom names and descriptions. It also allows you to choose if a responder type should be a one person role or a multi person role.

Reviewer: ```suggestion
If you wish to create custom responder roles, you can do so in the [Incident Settings for Responder Types][15]. This allows you to create new responder types with custom names and descriptions. It also allows you to choose if a responder type should be a one person role or a multi person role.
```

> **Note:** These roles are unrelated to those found in the [Role Based Access Control (RBAC)][14] system. RBAC roles control a user's permissions to access certain features in Datadog. The Responder Types system in Incident Management does not change a user's permissions in any capacity. It is instead about inviting responders to your incidents and giving them a documented role in your response process for visibility. 

Reviewer: ```suggestion
**Note:** These roles are unrelated to those found in the [Role Based Access Control (RBAC)][14] system. RBAC roles control a user's permissions to access certain features in Datadog. The Responder Types system in Incident Management does not change a user's permissions in any capacity. It is instead about inviting responders to your incidents and giving them documented roles in your response process for visibility. 
```

> If you add an individual as a responder, they are notified through the email associated with their Datadog account. Anyone is able to change the role of a responder, but you can only remove an individual from an incident's Response Team if they have the general `Responder` role assigned and have no activity in the incident. If there is already an `Incident Commander` assigned to an incident, assigning another individual as the `Incident Commander` transfers  that role over to them. The previous `Incident Commander` is reassigned the general `Responder` role. A similar reassignment will happen whenever you reassign one of your custom one person roles.

Reviewer: ```suggestion
If you add an individual as a responder, they are notified through the email associated with their Datadog account. Anyone is able to change the role of a responder, but you can only remove an individual from an incident's Response Team if they have the general `Responder` role assigned and have no activity in the incident. If there is already an `Incident Commander` assigned to an incident, assigning another individual as the `Incident Commander` transfers that role over to them. The previous `Incident Commander` is reassigned the general `Responder` role. A similar reassignment happens whenever you reassign one of your custom one person roles.
```
